CHANGELOG.md

```diff
@@ -1,6 +1,10 @@
 CHANGELOG
 =========
 
+## HEAD (Unreleased)
+
+- Update SummaryEvent to use the actual name plus path for Local Policy Packs
```

Comment:
Suggestion: "Update `SummaryEvent` to include the actual name and local file path for locally-executed policy packs."